### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qingqing01**

### CODE REVIEW

File: PaddleCV/PaddleDetection/README.md [hunk 1]
```diff
@@ -13,23 +13,26 @@ flexible, catering to research needs.
 
 ## Introduction
 
-Design Principles:
+Major features:
```
Comment (willthefrog) [inline PaddleCV/PaddleDetection/README.md]: simply `Features`?

Comment (heavengate) [reply PaddleCV/PaddleDetection/README.md]: features -> Features

Comment (qingqing01) [reply PaddleCV/PaddleDetection/README.md]: Done.

File: PaddleCV/PaddleDetection/docs/GETTING_STARTED.md [hunk 2]
```diff
@@ -75,8 +75,13 @@ path, simply add a `--save_file=` flag.
...
 
+**Q**: Why do I get `NaN` loss values during single GPU training?
+**A**: The default learning rate is tuned to multi-GPU training (8x GPUs), it must
+be adapted for single GPU training accordingly (e.g., divide by 8).
 
-Q: Why do I get `NaN` loss values during single GPU training?
 
-A: The default learning rate is tuned to multi-GPU training (8x GPUs), it must
-be adapted for single GPU training accordingly (e.g., divide by 8).
+**Q**: How to further reduce GPU memory?
```
Comment (willthefrog) [inline PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: **Q**: How to reduce GPU memory usage?
**A**: Setting environment variable `FLAGS_conv_workspace_size_limit` to a smaller number can reduce GPU memory footprint without affecting training speed.
Take Mask-RCNN (R50) as example, by setting `export FLAGS_conv_workspace_size_limit=512`, batch size could reach 4 per GPU.

Comment (qingqing01) [reply PaddleCV/PaddleDetection/docs/GETTING_STARTED.md]: Done. Thanks!

**Review by heavengate — COMMENTED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Pull request merged**